### DISCUSSION TITLE
What's newer 0.30.2 or 0.31.1 ?

### DISCUSSION THREAD

**Discussion by vvscode** [react-native-maps/react-native-maps#4324] [Q&A] [answered]
Hey

I'm a bit confused with the order of releases  (don't want to migrate to the 1.0.0 yet)
![image](https://user-images.githubusercontent.com/6904368/178929510-b7e9cb61-8ec7-4dca-8ca5-67f91dd416de.png)

Which one is newer? Is it valid?

**Comment by vvscode** [comment]
I can guess probably from 
```
[!] CocoaPods could not find compatible versions for pod "Google-Maps-iOS-Utils":
  In snapshot (Podfile.lock):
    Google-Maps-iOS-Utils (= 3.10.3)

  In Podfile:
    Google-Maps-iOS-Utils

    react-native-google-maps (from `../node_modules/react-native-maps`) was resolved to 0.31.1, which depends on
      Google-Maps-iOS-Utils (= 4.1.0)
```

But I'd like to clarify the situation as Changelog is not updated

**Comment by vvscode** [comment]
Ok, 0.30.x is a separate branch, while 0.31 is master

**Comment by monholm** [comment, marked as answer]
Hi @vvscode 👋

`v0.30.2` was released to maintain expo users stuck on the `v0.30.x` version. I put quite a bit of effort into staying compatible with semantic versioning, meaning `v0.30.2` would never include anything not included in `v0.31.1`. Github order releases based on release date, so don't put too much into the order of which they are displayed :)
